src/KeyVault/KeyVault/help/Restore-AzKeyVaultSecurityDomainBlob.md

```diff
@@ -0,0 +1,211 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.KeyVault.dll-Help.xml
+Module Name: Az.KeyVault
+online version: https://learn.microsoft.com/powershell/module/az.keyvault/restore-azkeyvaultsecuritydomainblob
+schema: 2.0.0
+---
+
+# Restore-AzKeyVaultSecurityDomainBlob
+
+## SYNOPSIS
+Restores an exported managed HSM locally to get ready for importing.
+
+## SYNTAX
+
+```
+Restore-AzKeyVaultSecurityDomainBlob -Keys <KeyPath[]> -ExchangeKey <String> -SecurityDomainPath <String>
```

Comment:
To align with Export-.SecurityDomain, I prefer to use Export-AzKeyVaultSecurityDomainBlob. The naming of noun also needs to discuss.